app/routes.js

```diff
@@ -25,18 +25,12 @@ export const routes = (authProvider) => {
 
   router.use('/health', healthRoutes);
 
-  router.get('/login', authProvider.authenticate({
-    successReturnToOrRedirect: '/',
-  }));
+  router.get('/login', authProvider.login());
 
-  router.get('/oauth/callback', authProvider.authenticate({
-    callback: true,
-    successReturnToOrRedirect: '/',
-    failureRedirect: '/',
-  }));
+  router.get('/oauth/callback', authProvider.loginCallback());
 
   router.get('/logout', async (req, res) => {
-    const url = await authProvider.logout();
+    const url = await authProvider.logout({ req });
```

Comment:
Do we need to pass in the `{req}` the `logout` function doesn't seem to have it defined

Reply:
nope - deleted